test/helpers/user.ts

```diff
@@ -1,78 +1,85 @@
 import { createUser } from "@app/users"
 import { yamlConfig } from "@config"
 import { CouldNotFindUserFromPhoneError } from "@domain/errors"
-import {
-  AccountsRepository,
-  UsersRepository,
-  WalletsRepository,
-} from "@services/mongoose"
+import { AccountsRepository, UsersRepository } from "@services/mongoose"
 import { User } from "@services/mongoose/schema"
 import { toObjectId } from "@services/mongoose/utils"
 
 const users = UsersRepository()
 
-export const getPhoneByTestUserIndex = (index: number) => {
-  const entry = yamlConfig.test_accounts[index]
+const getPhoneByTestUserIndex = (ref: string) => {
+  const entry = yamlConfig.test_accounts.find((item) => item.ref === ref)
   const phone = entry.phone as PhoneNumber
   return phone
 }
 
-export const getUserByTestUserIndex = async (index: number) => {
-  const phone = getPhoneByTestUserIndex(index)
+const getUserByTestUserRef = async (ref: string) => {
+  const phone = getPhoneByTestUserIndex(ref)
   const user = await UsersRepository().findByPhone(phone)
   if (user instanceof Error) throw user
   return user
 }
 
-export const getAccountByTestUserIndex = async (index: number) => {
-  const user = await getUserByTestUserIndex(index)
+export const getAccountByTestUserRef = async (ref: string) => {
+  const user = await getUserByTestUserRef(ref)
   const account = await AccountsRepository().findByUserId(user.id)
   if (account instanceof Error) throw account
   return account
 }
 
-export const getUserIdByTestUserIndex = async (index: number) => {
-  const user = await getUserByTestUserIndex(index)
+export const getUserIdByTestUserRef = async (ref: string) => {
+  const user = await getUserByTestUserRef(ref)
   return user.id
 }
```

Comment:
what is the value if having this function versus just getting the user and accessing the id property?

Reply:
several of the functions seem to fall into this category

Reply:
diff looks like this: https://github.com/GaloyMoney/galoy/pull/1043

not sure I'm a huge fan of it. I don't like doing `()` around the `await`

Reply:
thats fair, it does look cleaner the way it is